ct_core/include/ct/core/systems/discrete_time/linear/DiscreteLTISystem.h

```diff
@@ -0,0 +1,234 @@
+/**********************************************************************************************************************
```

Comment:
the rest looks good to me! There is only minor formatting issues, I believe, but it is going to be fixed the next time clang-format is ran over the entire project.